testdata/statement/remove.go.13.go

```diff
@@ -0,0 +1,52 @@
+// +build example-main
+
+package example
+
+import "fmt"
+
+func foo() int {
+	n := 1
+
+	for i := 0; i < 3; i++ {
+		if i == 0 {
+			n++
+		} else if i == 1 {
+			n += 2
+		} else {
+			n += 3
+		}
+
+		n++
+	}
+
+	if n < 0 {
+		n = 0
+	}
+
+	n++
+
+	n += bar()
+
+	bar()
+	bar()
+
+	switch {
+	case n < 20:
+		n++
+	case n > 20:
+		n--
+	default:
+		n = 0
+		_, _ = fmt.Println, n
```

Comment:
This is a good example for the problem of https://github.com/zimmski/go-mutesting/pull/20/files#r68056497